src/Detector/Node/NodeDetector.cs

```diff
@@ -212,14 +217,20 @@ private IEnumerable<FrameworkInfo> DetectFrameworkInfos(DetectorContext context)
                 // wild-card dependency
                 (bool isWildCardDependency, string wildCarddependencyName) = GetWildCardDependency(dependencyName);
```

Comment:
Nit: not blocking, but an uppercase is missing in `wildCarddependencyName`

Reply:
Ah yes, updated. I will need a new approval :)